doc/design/save_model_metadata.md

```diff
@@ -0,0 +1,67 @@
+# Save the Model Metadata After Training
+
+## Background: Why to Save the Model Metadata
+
+Currently, we have saved the model structure and weights after training in the SQLFlow. However, there are some requirements that we have to save the model metadata too:
+
+- We should save the docker image name which is used in the `TO TRAIN` statement so that we can use the same docker image in the `TO PREDICT` statement.
+
+  Users can specify the docker image name to run the `TO TRAIN` statement like this:
+
+  ```sql
```

Comment:
Can put the example at the beginning, then explain the problem we met.

Reply:
Done.